mta-sts.md

```diff
@@ -342,25 +299,16 @@ An example control flow for a compliant sender consists of the following steps:
 1. Check for a cached, non-expired policy. If none exists, attempt to fetch a
```

Comment:
> 1. Check for a cached, non-expired policy
The statement is not clear on whether the expiry is based on max-age in the json policy file or DNS.